styleguide/derek/_partials/_floor.ejs

```diff
@@ -0,0 +1,4 @@
+<%- partial("_backtotop") %>
+<%- partial("_rug") %>
+<%- partial("_footer") %>
+<%- partial("_basement") %>
```

Comment:
Created this to be a much easier way to include all our bottom of the page elements